src/modal/modal.stories.ts

```diff
@@ -1,12 +1,28 @@
 import { html, TemplateResult } from 'lit';
-import './index.js';
 import { Meta } from '@storybook/web-components';
+import './index.js';
+import '@tapsioss/icons/dist/icons/circle-check-fill'
```

Comment:
Please apply the ESLint fix command to this file

```suggestion
import '@tapsioss/icons/dist/icons/circle-check-fill';
```